content/blog/2025/07/kubewarden-1.27-release.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Kubewarden 1.27 Release"
+authors:
+  - José Guilherme Vanz
+date: 2025-07-23
+---
+
+Here's a look at the key updates and improvements in the latest release.
+
+## New High-Risk Service Account Policy
+
+In this release, we've introduced a new policy to improve cluster security. The
+**High-Risk Service Account Blocker** policy, as its name suggests, blocks
+workloads that attempt to run with a service account that has excessive
+permissions.
+
+This policy allows cluster operators to define a list of forbidden permissions.
+Workloads using a service account with any of these permissions will be
+blocked. Check out the policy's
+[repository](https://github.com/kubewarden/high-risk-service-account-policy) to
+learn more about its configuration.
+
+## New _kubernetes/can-i_ Host Capability
+
+To make the High-Risk Service Account Blocker policy more efficient, this
+release introduces a new host capability:
+[`kubernetes/can_i`](https://docs.kubewarden.io/reference/spec/host-capabilities/kubernetes#operation---can_i).
+
+This capability leverages the Kubernetes SubjectAccessReview API to check if a
+service account has permission to perform certain operations. This approach
+avoids the need to fetch numerous RBAC resources to determine if the service
+account can perform the blocked operations, making the check much faster and
+more reliable.
+
+## _kwctl_ Now Mirrors _policy-server_ Behavior
```

Comment:
We should mention which behaviours are those: policy mode (accepting and printing a warning when on `monitor` mode), honouring the custom rejection message, rejecting and printing an error if a policy mutates a request but doesn't declare it is a mutating policy. It would be nice to have small 2 liner examples showcasing the output.